libs/coin-evm/src/__tests__/unit/api/explorer/etherscan.unit.test.ts

```diff
@@ -122,7 +126,7 @@ describe("EVM Family", () => {
               ethereumLikeInfo: {
                 chainId: 1,
                 // no explorer
-              },
+              } as EthereumLikeInfo,
```

Comment:
It's needed because our `ethereumLikeInfo` object is misconfigured for the purpose of this test and does not have a `node` property provided, which is required by `EthereumLikeInfo` type.

If this is removed, you get a TS error stating that a `node` property is required, preventing us to test this scenario.

That's what we are testing in this test "should throw an error if the currency is misconfigured"